scripts/minikube.sh

```diff
@@ -104,11 +112,25 @@ up)
         install_kubectl
     fi
 
-    enable_psp
+    MINIKUBE_MAJOR=$(minikube_version 1)
+    MINIKUBE_MINOR=$(minikube_version 2)
+    MINIKUBE_PATCH=$(minikube_version 3)
 
     echo "starting minikube with kubeadm bootstrapper"
-    # shellcheck disable=SC2086
-    minikube start --memory="${MEMORY}" -b kubeadm --kubernetes-version="${KUBE_VERSION}" --vm-driver="${VM_DRIVER}" --feature-gates="${K8S_FEATURE_GATES}" ${EXTRA_CONFIG}
+    if [[ "${MINIKUBE_MAJOR}" -ge 1 ]] && [[ "${MINIKUBE_MINOR}" -ge 11 ]] && [[ "${MINIKUBE_PATCH}" -ge 1 ]] || [[ "${MINIKUBE_MAJOR}" -ge 1 ]] && [[ "${MINIKUBE_MINOR}" -ge 12 ]]; then
+        enable_psp
+        # shellcheck disable=SC2086
```

Comment:
What is being reported here? Please leave a comment when disabling a shellcheck.

Reply:
not added in this one, it just got moved from line 110